### ISSUE TITLE
Error: This app tried to connect to a canister on your behalf without the proper permissions

### ISSUE THREAD

**Issue by honopu** [Psychedelic/plug#575]
```
1. window.ic.plug.requestConnect();
2. window.ic.plug.createAgent({ whitelist: ["rd6wb-lyaaa-aaaaj-acvla-cai"],  host: "https://ic0.app" })
3. window.ic.plug.createActor({ canisterId: xxxx,  interfaceFactory: xxx });
```
Then call transfer method and throw error:
`
This app tried to connect to a canister (rd6wb-lyaaa-aaaaj-acvla-cai) on your behalf without the proper permissions. Please contact this project’s developers and share the error with them so they can fix it.
`
So the whitelist passed by createAgent is not working right now?

![image](https://user-images.githubusercontent.com/85238703/187071571-59596726-5c83-4ef6-973c-78e500a3c3e4.png)
<img width="1090" alt="image" src="https://user-images.githubusercontent.com/85238703/187071581-8b582044-e487-4cae-a707-a2b0bc1915e7.png">

**Comment by ICDinosDevelopers** [comment]
I have exactly the same problem since yesterday. If you know how to solve this problem please let me know!

**Comment by Werwolf888** [comment]
I have exactly the same problem since yesterday. If you know how to solve this problem please let me know!

**Comment by honopu** [comment]
> I have exactly the same problem since yesterday. If you know how to solve this problem please let me know!

You can use `window.ic.plug.requestConnect({ whitelist: ["all your canister ids"] })`

**Comment by riveign** [comment]
Hi! This was fixed in the release 0.5.4.1 https://github.com/Psychedelic/plug/releases/tag/V0.5.4.1 closing the issue :D
